### PR TITLE
wptserve: reduce .headers clutter

### PR DESCRIPTION
**Pull request by annevk**

### CODE REVIEW

File: tools/wptserve/wptserve/handlers.py [hunk 1]
```diff
@@ -95,15 +95,29 @@ def list_items(self, base_path, path):
             link = urljoin(base_path, "..")
             yield ("""<li class="dir"><a href="%(link)s">%(name)s</a></li>""" %
                    {"link": link, "name": ".."})
+        items = []
+        last_item = None
```
Comment (jgraham) [inline tools/wptserve/wptserve/handlers.py]: Can we call this `prev_item` instead? At first I thought it was "last" as in "last in the list".

Comment (annevk) [reply tools/wptserve/wptserve/handlers.py]: We totally can, but is that really all the feedback?

Comment (jgraham) [reply tools/wptserve/wptserve/handlers.py]: Did you want more feedback? :p Apart from that it seems reasonable to land the change afaict.

Comment (annevk) [reply tools/wptserve/wptserve/handlers.py]: I was expecting my Python to not be up to snuff.

**Review by jgraham — APPROVED** [review]

**Pull request merged**